src/clj/witan/send/model/prepare.clj

```diff
@@ -0,0 +1,217 @@
+(ns witan.send.model.prepare
+  (:require [clojure.core.matrix.dataset :as ds]
+            [witan.send.states :as states]
+            [witan.send.utils :as u]
+            [medley.core :as medley]
+            [witan.send.params :as p]
+            [witan.send.check-inputs :refer [run-input-checks]]
+            [clojure.set :refer [rename-keys]]
+            [clojure.walk :refer [postwalk]]
+            [witan.send.report :as report]
+            [schema.core :as s]
+            [witan.send.schemas :as sc]))
+
+(defn int-ceil [n]
```

Comment:
Still not getting rid of this or building into where it is used?

Reply:
just this mechanical bit as a first pass. We an kill it off later when we look at it. (iterateiterateiterate)